docs/releases/v2023.2.rst

```diff
@@ -0,0 +1,218 @@
+Gluon 2023.2
```

Comment:
Mention the issue with Ubiquiti Flash and ask for help

Reply:
@herbetom mentioned in the meetup that they have a device available for opening and fixing this. But it might still be useful to ask for further devices for testing the final fix.